dapr-http-client/src/Dapr/Client/HttpClient/Req.hs

```diff
@@ -24,8 +24,9 @@ makeHttpRequest ::
 makeHttpRequest config method subUrl reqBody responseHandler options = runReq defaultHttpConfig $ do
   let host = daprHost config
       apiVersion = daprApiVersion config
+      defaultContentType = header "Content-Type" "application/json"
```

Comment:
Should we make `"Content-Type": "application/json"` as default header? Since it's for **PUT** and **POST** request only, **GET** and **DELETE** request shouldn't have content-type in header. An option is to check the `method` before setting Content-Type

Reply:
Implemented this. Thanks for your help here.